src/components/Footer.jsx

```diff
@@ -43,76 +43,36 @@ class SiteFooter extends React.Component {
       intl,
     } = this.props;
     const showLanguageSelector = supportedLanguages.length > 0 && onLanguageSelected;
-    const config = getConfig();
+    //const { config } = this.context;
```

Comment:
nit: remove this. Generally speaking, we should probably fix test failures above and below.